app/controllers/schools_controller.rb

```diff
@@ -165,66 +165,48 @@ def authorized_students_for_overview(school)
     end
   end
 
-  # Methods for Dashboard
-
-  def individual_student_dashboard_data(student)
-    # This is a temporary workaround for New Bedford
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
-      first_name: student.first_name,
-      last_name: student.last_name,
-      id: student.id,
-      grade: student.grade,
-      race: student.race,
-      homeroom_label: homeroom_label,
-      absences: student.dashboard_absences,
-      tardies: student.dashboard_tardies,
-      event_notes: student.event_notes
-    })
-  end
+  # Methods for Dashboards #
 
   def individual_student_absence_data(student)
-    # Gathers only the information needed for a specific dashboard view.
-    homeroom_label = student.try(:homeroom).try(:educator).try(:full_name) || student.try(:homeroom).try(:name)
-    HashWithIndifferentAccess.new({
+    {
```

Comment:
I believe `HashWithIndifferentAccess` is slower than just a plain hash and uses more memory. (Here's one person's benchmark data: https://gist.github.com/tiagoamaro/c82a27aceedfc901b081#file-results_with_4-2-5-txt.) Since swapping it out for a plain hash didn't cause any feature regression that I could see, I figured why not.